test/k8sT/Policies.go

```diff
@@ -591,6 +605,82 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 			testConnectivitytoRedis()
 		})
 	})
+
+	Context("KubernetesNetworkPolicy between server and client", func() {
+		BeforeEach(func() {
+			kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+
+			By("Creating the namespace that will be used for the pods")
+			_, err := kubectl.CoreV1().Namespaces().Create(&v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}})
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Creating a simple server that serves on port 80 and 81.")
+			podServer, service = createServerPodAndService(kubectl, namespace, "server", []int{80, 81})
+
+			By("Waiting for pod ready", func() {
+				ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+				defer cancel()
+				err := kubectl.WaitForPodReady(ctx, namespace, podServer.Name)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			By("Testing pods can connect to both ports when no policy is present.")
+			testCanConnect(kubectl, namespace, "client-can-connect-80", service, 80, true)
+			testCanConnect(kubectl, namespace, "client-can-connect-81", service, 81, true)
+		})
+
+		AfterEach(func() {
+			cleanupServerPodAndService(kubectl, podServer, service)
+			By("Deleting the namespace that was used for the pods")
+			err := kubectl.CoreV1().Namespaces().Delete(namespace, &metav1.DeleteOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			if CurrentGinkgoTestDescription().Failed {
+				ciliumPod, _ := kubectl.GetCiliumPodOnNode(helpers.KubeSystemNamespace, helpers.K8s1)
+				kubectl.CiliumReport(helpers.KubeSystemNamespace, ciliumPod, []string{
+					"cilium bpf tunnel list",
```

Comment:
I don't see any real value to have in logs the `bpf tunnel list`. I would change this to `policy get`

Reply:
Done